docs/3.0.0-beta.x/plugins/upload.md

```diff
@@ -272,48 +272,87 @@ In our second example, you can upload and attach multiple pictures to the restau
 }
 ```
 
-## Install providers
+## Using a provider
```

Comment:
I have a question about the email provider, we changed configuration management with file configurations.
What about the email plugin. Because this plugin works exactly the same as upload plugin.
So I don't know if config has been updated also in email plugin.
If yes, should we update the doc too.
If no, can we implement it rapidly?

Reply:
The email plugin hasn't changed yet, hence the doc not being updated. It will be changed in an upcoming PR